GOVERNANCE.md

```diff
@@ -188,18 +189,18 @@ https://github.com/orgs/zarr-developers/teams/core-devs
 
 Decisions should be made in accordance with the mission and values of the Zarr project.
 
-Zarr uses a “consensus seeking” process for making decisions. The group
-tries to find a resolution that has no open objections among core developers.
-Core developers are expected to distinguish between fundamental objections to a
-proposal and minor perceived flaws that they can live with, and not hold up the
-decision-making process for the latter.  If no option can be found without
-objections, the decision is escalated to the SC, which will itself use
-consensus seeking to come to a resolution. In the unlikely event that there is
-still a deadlock, the proposal will move forward if it has the support of a
-simple majority of the SC.
+Zarr uses a “consensus-seeking” process for making decisions. The group tries to
+find a resolution that has no open objections among core developers and ZIC
```

Comment:
This feels overly specific. For example, just on the zarr-python repo I'm not sure the ZIC would be objecting, right?

Reply:
Yes! I think involving ZIC in the decision-making process for Zarr-Python doesn't feel right. This should be taken care of by [Zarr-Python Core Devs](https://github.com/orgs/zarr-developers/teams/python-core-devs).

I think ZIC should be majorly involved when changes are added to the specification.